### PR TITLE
feat: DAH-2741 assemble how to apply page

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Puts together the main text of the how to apply page.

Note for reviewer: I tried to write it so that it would be clear to understand and easy to edit if necessary! Pls give me feedback on the organization and the naming if you have anything! Also, don't worry too much about the spacing and matching the figma perfectly - There's a little flexibility in design (discussed with Yindi), so they are going to review that.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2741

### Version Control

- [ ] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [ ] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [x] all automated code checks pass (linting, tests, coverage, etc.)
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions work for PA testers
- [x] instructions have already been performed at least once

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/javascript/pages/how-to-apply/how-to-apply.tsx [hunk 1]
```diff
@@ -1,19 +1,231 @@
...
 const HowToApply = (_props: HowToApplyProps) => {
   return (
     <Layout title={t("pageTitle.howToApply")}>
-      <section className="bg-gray-300 flex justify-center">
-        <Heading> {t("pageTitle.howToApply")}</Heading>
+      <section className="flex flex-col justify-center md:px-5">
+        <article className="markdown max-w-5xl m-auto">
+          <div className="pt-4 md:py-0 max-w-3xl">
+            <div className="markdown-section markdown-section--with-padding">
+              <div className="markdown-section__inner">
```
[the remark this answers is not in the document]
Comment (tallulahkay) [reply app/javascript/pages/how-to-apply/how-to-apply.tsx:218]: I used it based on the code for https://housing.sanjoseca.gov/how-it-works: https://github.com/housingbayarea/bloom/blob/main/sites/public/src/pages/how-it-works.tsx

That's what Yindi referenced when making their design

File: app/javascript/pages/how-to-apply/how-to-apply.tsx [hunk 2]
```diff
@@ -1,19 +1,231 @@
...
+                <article className="markdown">
```
[the remark this answers is not in the document]
Comment (tallulahkay) [reply app/javascript/pages/how-to-apply/how-to-apply.tsx:219]: Not sure - based on code ref

File: app/javascript/pages/how-to-apply/how-to-apply.tsx [hunk 3]
```diff
@@ -1,19 +1,231 @@
...
+      </div>
+    </HowToApplyListItem>
+  )
+}
+
+const CreateBoxAccountStep = () => {
+  return (
+    <HowToApplyListItem headerText={t("howToApplyPage.howToApplySection.step4.title")}>
+      <div className="text-base pb-2">
+        {renderInlineMarkup(t("howToApplyPage.howToApplySection.step4.p1", { url: "#" }))}
```
[the remark this answers is not in the document]
Comment (tallulahkay) [reply app/javascript/pages/how-to-apply/how-to-apply.tsx:155]: No - I just made [this ticket](https://sfgovdt.jira.com/browse/DAH-2847) for Yindi and Bridget to fill out. I will add it in code comments

**Review by jimlin-sfgov — APPROVED** [review]
LGTM

good job organizing this content-heavy page!

**Pull request merged**